pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -401,17 +409,22 @@ func (og *operationGenerator) GenerateDetachVolumeFunc(
 
 	if volumeToDetach.VolumeSpec != nil {
 		// Get attacher plugin
+		nu, err := nodeUsingCSIPlugin(og, volumeToDetach.VolumeSpec, volumeToDetach.NodeName)
+		if err != nil {
+			return volumetypes.GeneratedOperations{}, volumeToDetach.GenerateErrorDetailed("DetachVolume.NodeUsingCSIPlugin failed", err)
+		}
+
 		// useCSIPlugin will check both CSIMigration and the plugin specific feature gate
```

Comment:
I don't see useCSIPlugin checking the plugin specific feature gate... am I missing it?

Reply:
each plugin specific feature gate is encoded by the plugin itself and is checked by: `vpm.IsPluginMigratableBySpec(spec)`

Reply:
ah, I see it now, thanks.